features/src/main/feature/camel-features.xml

```diff
@@ -765,6 +765,22 @@
         <bundle dependency='true'>mvn:commons-io/commons-io/2.15.1</bundle>
         <bundle>mvn:org.apache.camel.karaf/camel-exec/${project.version}</bundle>
     </feature>
+    <feature name='camel-elytron' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-core</feature>
+        <feature version='${project.version}'>camel-http</feature>
+        <feature>jakarta-servlet</feature>
+        <bundle dependency='true'>mvn:org.apache.camel.karaf/camel-undertow/${project.version}</bundle>
```

Comment:
AFAIU there is no direct dependency on `camel-http` but rather on `camel-undertow` and the feature `camel-http` has been added to have access to `camel-attachments` which is in fact a dependency of `camel-undertow` so here I suggest to add the feature `camel-undertow` and make it depends on this new feature. It can be done within the context of this PR for simplicity

Reply:
added camel-undertow feature and removed the camel-http.
However, camel-attachments is not a dependency of camel-undertow so I added only the necessary dependencies : 
`<bundle>mvn:org.apache.camel.karaf/camel-attachments/${project.version}</bundle>
        <bundle>mvn:org.apache.camel.karaf/camel-http-base/${project.version}</bundle>`

Reply:
good catch, I did not trust the camel pom enough.
Now undertow feature is more consistent, elytron just add the elytron dependencies